README.md

```diff
@@ -1 +1,19 @@
-# README
\ No newline at end of file
+# Northcoders News
+
+- Link to delpoyed app: https://rc-nc-news.netlify.app/
+
+- A Reddit-like website that interacts with a back-end server and database to serve news articles. The front-end React app allows interactivity such as sorting/voting on articles, logging in as a user and posting/deleting comments.
+
+- You must be a valid user to comment; try logging in as "grumpy19".
+
+- Here is the repo for the back-end server: https://github.com/robbochobbo/my-backend-project
+
+Required minimum versions:
+- Node.js v21.5.0
+
+To clone this repo:
+
+Fork on GitHub: https://github.com/robbochobbo/nc-news
```

Comment:
this line contradicts the next one a bit to me, I don't think you need to encourage them to fork it because they can just run it locally from the clone